components/org.wso2.carbon.identity.conditional.auth.functions.user/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/user/IsMemberOfAnyOfGroupsFunctionImpl.java

```diff
@@ -93,4 +168,49 @@ private List<String> getDomainAwareGroupNames(String userStoreDomain, List<Strin
         }
         return groupsWithDomain;
     }
+
+    /**
+     * Retrieve the groups claim configured for the federated IDP.
+     *
+     * @param user Authenticated user.
+     * @return  groups claim configured for the IDP.
+     */
+    private static String getGroupsClaimURIByClaimMappings(JsAuthenticatedUser user){
+
+        String groupsClaimURI = null;
```

Comment:
Do we need to initiate this variable?. We can return `groupClaimsURI` in the last if block.

Reply:
Resolved by https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/132/commits/475d6b6266ce5afbfd91f6a634df9e8586dbce07